src/calc.cc

```diff
@@ -26,7 +26,7 @@
 
 #include "calc.h"
 #include "grid.h"
-#include <cmath>
+#include <math.h>
```

Comment:
What does this solve? `cmath` is C++ standard

Reply:
Good point, not sure why I changed that, I think I was just confused.  I updated to use cmath again, and fixed a few other includes.